README.md

```diff
@@ -12,6 +12,17 @@ Monorepo to hold all shareable linting and formatting configurations.
 
 When using both ESLint and Prettier in a project, it's nice to have files automatically have ESLint errors fixed on save as well as have non JavaScript/TypeScript files formatted on save. This section explains how to set that up in an editor.
 
+### Versions Bump
+Patch releases: 
+  - Bump dependencies to patch or minor releases
+  - Fixs not impacting the configs or adding new rules
+Minor releases: 
+  - Adding exceptions to rules (ex: disable a default rule)
+  - Changin Minor rules (Nothing that will reformat code by default)
+Major releases: 
+  - Bump dependencies to major releases
+  - Changing rules that will reformat code by default (ex: `{ printWidth: 80 }` -> `{ printWidth: 120 }`)
```

Comment:
Changes to lint rules shouldn't cause major version bumps because they won't break anything when used in other apps. Bumping major versions of dependencies should, however, result in a new major release version.

Reply:
in our case it would be `KIND OF` a breaking change as the linter+prettifier would start  major formating in any PR as soon as released... I knew this could be a friction point. 
My idea Was that we want to know if a major reformating rule is released a chose when to bump the Repo, instead of automatically start scrapping PR's visibility. But I could be wrong

Reply:
"Major reformating" doesn't mean anything is breaking and then we would have to question what is considered a "major reformating"?